Changelog.md

```diff
@@ -1,5 +1,90 @@
 ## Ignition Msgs 8.x
 
+### Ignition Msgs 8.3.0 (2022-03-24)
+1. Merge pull request #226 from ignitionrobotics/merge78
+    * [Pull request #226 from ignitionrobotics/merge78](https://github.com/ignitionrobotics/ign-msgs/pull/226 from ignitionrobotics/merge78)
+
+1. Merge branch 'ign-msgs7' into merge78
+    * [Pull request #Merge branch 'ign-msgs7' into merge78](https://github.com/ignitionrobotics/ign-msgs/pull/Merge branch 'ign-msgs7' into merge78)
+
+1. Prepare for 7.3.0 release
+    * [Pull request #234](https://github.com/ignitionrobotics/ign-msgs/pull/234)
+
+1. Merge pull request #233 from scpeters/merge57
+    * [Pull request #233 from scpeters/merge57](https://github.com/ignitionrobotics/ign-msgs/pull/233 from scpeters/merge57)
+
+1. Merge branch 'ign-msgs5' into merge57
+    * [Pull request #Merge branch 'ign-msgs5' into merge57](https://github.com/ignitionrobotics/ign-msgs/pull/Merge branch 'ign-msgs5' into merge57)
+
+1. 🎈 5.9.0
+    * [Pull request #232](https://github.com/ignitionrobotics/ign-msgs/pull/232)
```

Comment:
please remove all merge commits and release PRs from the changelog

Reply:
Done